mmedit/models/backbones/encoder_decoders/decoders/plain_decoder.py

```diff
@@ -1,6 +1,69 @@
+import warnings
+
 import torch.nn as nn
+import torch.nn.functional as F
 from mmcv.cnn.utils.weight_init import xavier_init
 from mmedit.models.registry import COMPONENTS
+from torch.autograd import Function
+from torch.nn.modules.pooling import _MaxUnpoolNd
+from torch.nn.modules.utils import _pair
+
+
+class MaxUnpool2dop(Function):
+    """We warp the `torch.nn.functional.max_unpool2d`
+    with an extra `symbolic` method, which is needed while exporting to ONNX.
+    Users should not call this function directly.
+    """
+
+    @staticmethod
+    def forward(ctx, input, indices, kernel_size, stride, padding,
+                output_size):
+        return F.max_unpool2d(input, indices, kernel_size, stride, padding,
+                              output_size)
+
+    @staticmethod
+    def symbolic(g, input, indices, kernel_size, stride, padding, output_size):
+        warnings.warn(
```

Comment:
Docstring should be also added to these functions.

Reply:
> Docstring should be also added to these functions.

Maybe we do not need to add docstring for this function, as this function can not be called by users, but only be invoked by Pytorch engine while exporting to ONNX.

Reply:
All right.